### DISCUSSION TITLE
The API For transfering a slice indictates it's a certificate we transfer.

### DISCUSSION THREAD

**Discussion by Soped42** [project-origin/registry#95] [Q&A]
Implementation wise we are never transfering certificates but rather slices of quantities of energy.
However the API is as follows

TransferCertificate(FederatedCertifcateId, ShieldedValue, Key, PublicKey)
This is used to transfer a [Granular Certificate slice](https://project-origin.github.io/registry/concept/granular-certificates/readme.html#slices) from the current owner to a new owner.

Would it be better if it had slice in the API call, to indicate that we actually transfer a slice and not the certificate?

TransferCertificate**Slice**(FederatedCertifcateId, ShieldedValue, Key, PublicKey)
This is used to transfer a [Granular Certificate slice](https://project-origin.github.io/registry/concept/granular-certificates/readme.html#slices) from the current owner to a new owner.

**Comment by lauranolling** [comment]
Tagging @wisbech and @MartinSchmidt, as they may have inputs/opinions for this. :-)

**Reply by wisbech** [reply]
Dependent on the urgency of clarification of this matter I would propose to raise this question during the workshop the 20. and 21 of April at Energinet 👍

**Comment by MartinSchmidt** [comment]
Create issue from this discussion, the change will be made, and can be tracked in the issue #109
